createSpotifyPlaylist.py

```diff
@@ -0,0 +1,256 @@
+timeline and create playlists by checking @mentions and using those as search querys. This will
+require both the spotify and twitter api.
+"""
+import os
+import time
+import spotipy
+from spotipy.oauth2 import SpotifyOAuth
+
+
+
+class CreateSpotifyPlaylist:
+    """
+    A class used to communicate with the Spotify api.
+
+    Attributes
+    ----------
+    popular_tracks : list
+        A list of popular tracks on spotify
+
+    Methods
+    -------
+    get_spotify_playlists(self, query:str)
+        Searches spotify for playlists
+
+    get_popular_songs(self, spotify_playlist_id:str):
+        Gets most popular songs from spotify playlist
+
+    create_spotify_playlist(self, playlist_name:str, for_user:str)
+        Creates a new spotify playlists using a name and a users name
+
+    add_tracks_to_playlist(self, playlist_id:str)
+        Adds song to a spotify playlists
+    """
+
+    def __init__(self):
+        """
+        Init function
+        This function will call the class function `auth_spotify` and create a list for later use.
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        None
+        """
+        self.spotify_client = self.auth_spotify()
+        self.popular_tracks = []
+
+
+    @classmethod
+    def auth_spotify(cls):
+        """
+        Function to authenticate spotify
+        This function will get the spotify client so we can use the api by authenticating. It will
+        also set the scopes so we are able to use the tools that we need such as
+        `playlist-modify-public` to create public spotify playlist
+
+        Parameters
+        ----------
+
+        Returns
+        -------
+        str
+            The spotify client
+        """
+        scope = "user-library-read,user-top-read,playlist-modify-public"
+        spotify_client = spotipy.Spotify(auth_manager=SpotifyOAuth(scope=scope))
+        return spotify_client
+
+
+    def get_spotify_playlists(self, query: str) -> None:
+        """
+        Function to search spotify for playlists
+        This function will search spotify for playlists with the given query. For every playlist
+        we will get its most popular songs to do that we call the function `get_popular_songs`.
+        This function has no returns.
+
+        Parameters
+        ----------
+        query : str
+            The search query to find spotify playlist
+
+        Returns
+        -------
+        None
+        """
+        # search for playlist that match the query on spotify
+        playlists =  self.spotify_client.search(q=query, type='playlist', limit=5)
+
+        # find the most popular songs in each playlist
+        for playlist in playlists['playlists']['items']:
+            playlist_uri = playlist['uri']
+            self.get_popular_songs(playlist_uri)
+
+
+    def get_popular_songs(self, spotify_playlist_id:str) -> None:
+        """Function to get most popular songs from a spotify playlist
```

Comment:
put the three quotes `"""` on their own line - be consistent with the other docstring

Reply:
I have put the quotes on their own line